### DISCUSSION TITLE
Scss use #{!important} error

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#3892] [Help] [answered]
How to use `impatrant` in scss？

![QQ20210401-192901](https://user-images.githubusercontent.com/47672657/113287801-a5831580-9320-11eb-8e08-f7ede1522458.png)

**Comment by olets** [comment, marked as answer]
You can use an `important` variant. The docs explain how to build that exact one https://tailwindcss.com/docs/plugins#complex-variants. Of there's almost always a solution that doesn't require important 😄
